packages/clerk-js/src/ui/components/UserProfile/ProfilePage.tsx

```diff
@@ -17,6 +17,8 @@ export const ProfilePage = withCardStateProvider(() => {
   const { first_name, last_name } = useEnvironment().userSettings.attributes;
   const showFirstName = first_name.enabled;
   const showLastName = last_name.enabled;
+  const userFirstName = user.firstName || '';
+  const userLastName = user.lastName || '';
```

Comment:
Lets inline all these variables as they don't really benefit as much, eg 
```suggestion
  const { first_name: firstName, last_name: lastName } = useEnvironment().userSettings.attributes;
  
  ...
  // use firstName.enable, firstName.required and user.firstName directly
```

Reply:
@nikosdouvlis we need to have empty strings as default values(e.x. `  const userLastName = user.lastName || '';`) in order to make proper check later on(e.x. `lastNameField.value !== userLastName`).